XRTK-Core/Packages/com.xrtk.core/Services/BoundarySystem/MixedRealityBoundarySystem.cs

```diff
@@ -323,7 +323,7 @@ private GameObject BoundaryVisualizationParent
                 }
 
                 var visualizationParent = new GameObject("Boundary System Visualizations");
-                visualizationParent.transform.parent = MixedRealityToolkit.CameraSystem?.CameraRig.PlayspaceTransform;
+                visualizationParent.transform.SetParent(MixedRealityToolkit.CameraSystem?.MainCameraRig.PlayspaceTransform, false);
```

Comment:
We should check if the camera system is null here and fall back to `CameraCache.Main.parent`